Test/Utils/MultiNode/ContrailMultiNodeProvisioning.ps1

```diff
@@ -0,0 +1,73 @@
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testbed.ps1
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testenv.ps1
+. $PSScriptRoot\..\..\PesterLogger\PesterLogger.ps1
+. $PSScriptRoot\..\ComponentsInstallation.ps1
+
+# Import order is chosen explicitly because of class dependency
+. $PSScriptRoot\..\ContrailNetworkManager.ps1
+. $PSScriptRoot\..\..\..\CIScripts\Testenv\Testenv.ps1
+. $PSScriptRoot\MultiNode.ps1
+
+function New-MultiNodeSetup {
+    Param (
+        [Parameter(Mandatory=$true)] [string] $TestenvConfFile
+    )
+
+    $VMs = Read-TestbedsConfig -Path $TestenvConfFile
+    $OpenStackConfig = Read-OpenStackConfig -Path $TestenvConfFile
+    $ControllerConfig = Read-ControllerConfig -Path $TestenvConfFile
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute(
```

Comment:
Do we really need this message suppression?

Reply:
No. I'll remove it.